WordPress/Classes/Utility/Editor/GutenbergSettings.swift

```diff
@@ -51,6 +53,37 @@ class GutenbergSettings {
         }
     }
 
+    func performGutenbergPhase2MigrationIfNeeded() {
+        guard let account = AccountService(managedObjectContext: context).defaultWordPressComAccount() else {
+            return
+        }
+        var rollout = GutenbergRollout(database: database)
+        if rollout.shouldPerformPhase2Migration(userId: account.userID.intValue) {
+            setGutenbergEnabledForAllSites()
+            rollout.isUserInRolloutGroup = true
+        }
+    }
+
+    private func setGutenbergEnabledForAllSites() {
+        let allBlogs = BlogService(managedObjectContext: context).blogsForAllAccounts()
+        allBlogs.forEach { blog in
+            if blog.editor == .aztec {
+                setShowPhase2Dialog(true, for: blog)
+                database.set(true, forKey: Key.enabledOnce(for: blog))
+            }
+        }
+        let editorSettingsService = EditorSettingsService(managedObjectContext: context)
+        editorSettingsService.migrateGlobalSettingToRemote(isGutenbergEnabled: true, overrideRemote: true)
```

Comment:
I'm not seeing it, I think we're missing  the tracking code `trackSettingChange(true, "on_progressive_rollout_phase_2")`

Event param names defined here: https://github.com/wordpress-mobile/WordPress-Android/blob/0846701804aee3c1967a58c3065dc3a74e19bded/WordPress/src/main/java/org/wordpress/android/util/analytics/AnalyticsUtils.java#L77-L78

Reply:
Good catch! Where exactly is the tracks event being triggered on Android?

Reply:
Added! 👍